src/Umbraco.Web.Common/Routing/PublishedRouter.cs

```diff
@@ -0,0 +1,766 @@
+using System;
+using System.Linq;
+using System.Threading;
+using System.Globalization;
+using System.IO;
+using Umbraco.Core;
+using Umbraco.Core.Configuration.UmbracoSettings;
+using Umbraco.Core.Logging;
+using Umbraco.Core.Models;
+using Umbraco.Core.Models.PublishedContent;
+using Umbraco.Core.Services;
+using Umbraco.Web.Routing;
+using Umbraco.Web.Security;
+namespace Umbraco.Web.Common.Routing
+{
+    /// <summary>
+    /// Provides the default <see cref="IPublishedRouter"/> implementation.
+    /// </summary>
+    public class PublishedRouter : IPublishedRouter
```

Comment:
I assume this file was just moved? But i don't see it being deleted anywhere either so not sure if its changed?

Reply:
Don't know what happened here. We already have PublishedRouter in Umbraco.Core, so I will delete this.